### PR TITLE
fix: olm pinned (was master and giving problems)

### PR DESCRIPTION
**Pull request by Morriz**

### CODE REVIEW

File: schemas/gen-missing-crd-schemas.sh [hunk 1]
```diff
@@ -3,27 +3,28 @@
...
-rm -rf ./$gen_folder/*
+#rm -rf ./$gen_folder/*
 mkdir $input_folder >/dev/null
 rm -rf $input_folder/*
 
 # match all the crds of charts that didn't ship crds (some operators don't), and pull them
 # (expects kube context to have a cluster that has them all)
-for pkg in "argoproj" "cert-manager" "istio" "jaeger" "kiali" "knative"; do
+# for pkg in "argoproj" "operators.coreos" "cert-manager" "istio" "jaeger" "kiali" "knative"; do
+for pkg in "operators.coreos"; do
```
Comment (j-zimnowoda) [inline schemas/gen-missing-crd-schemas.sh:21]: Does it mean that we do not need to generate CRDs for `argoproj" "cert-manager" "istio" "jaeger" "kiali" "knative";` anymore ?

Comment (Morriz) [reply schemas/gen-missing-crd-schemas.sh:21]: no, this is a manual script, and the one using it should know what to do. If not all of those are deployed, but only one is, and that one needs to be added, this is the way to do it.

**Comment by staticvoid255** [comment]
Do you have any advice on how this should be tested? As in what is my starting point from which I can try to find issues?

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
olm cannot deploy during otomi installation
See more: https://github.com/redkubes/otomi-core/actions/runs/3648896922/jobs/6162870163

**Comment by Morriz** [comment]
> olm cannot deploy during otomi installation
> See more: https://github.com/redkubes/otomi-core/actions/runs/3648896922/jobs/6162870163

@j-zimnowoda I don't see any log output there. You have consistently pasted links to empty logs since a long time, so I ask you not to do that anymore. Or am I missing something?

**Comment by Morriz** [comment]
@j-zimnowoda OLM installs fine when I deploy the helm chart using this branch tag. Please report with proper reproduction instructions and logs next time.

**Comment by j-zimnowoda** [comment]
> > olm cannot deploy during otomi installation
> > See more: https://github.com/redkubes/otomi-core/actions/runs/3648896922/jobs/6162870163
> 
> @j-zimnowoda I don't see any log output there. You have consistently pasted links to empty logs since a long time, so I ask you not to do that anymore. Or am I missing something?

@Morriz if you look carefully then you will see 
image
The `Gather Otomi logs on failure` contain the logs from installation.

**Comment by Morriz** [comment]
Ah, ok. Is it possible to share deep links to that output?

**Comment by Morriz** [comment]
@j-zimnowoda I did fix the issue of deployment btw, so you can try install again.

**Comment by j-zimnowoda** [comment]
> Ah, ok. Is it possible to share deep links to that output?

Unfortunately not (or I do not know How to do it).

**Review by j-zimnowoda — APPROVED** [review]
The installation is successful

**Comment by Morriz** [comment]
@j-zimnowoda So no upgrade issues?

**Pull request merged**